txpool/account.go

```diff
@@ -44,7 +45,7 @@ func (m *accountsMap) exists(addr types.Address) bool {
 
 // getPrimaries collects the heads (first-in-line transaction)
 // from each of the promoted queues.
-func (m *accountsMap) getPrimaries() (primaries []*types.Transaction) {
+func (m *accountsMap) getPrimaries(baseFee uint64) (primaries []*types.Transaction) {
 	m.Range(func(key, value interface{}) bool {
 		addressKey, ok := key.(types.Address)
```

Comment:
As a side note, not related to this PR:
I think that we could just make type conversion of `value` parameter into `*account`, instead of calling 
`account := m.get(addressKey)` unnecesarily. WDYT, am I missing out something?

Reply:
that's a good point, i also think that conversion is a redundant overhead here